BibBuddy/app/src/main/java/de/bibbuddy/NoteDao.java

```diff
@@ -240,6 +240,16 @@ public String findTextById(Long id) {
     return noteText;
   }
 
+  /**
+   * This method gets text string of a specific note without formatting xml tags.
+   *
+   * @param id id of the note to look for
+   * @return returns the notes text value without formatting texts
+   */
+  public String findStrippedTextById(Long id) {
+    return findTextById(id).replaceAll("<[^>]+>", "");
```

Comment:
You should not replace all characters, because if the user uses some of those characters in the note, then this note's text value is not shown properly in the exported BibTeX-file. Maybe you should set limits for replaced chars.

Reply:
Hey, thanks for the suggestion! You're completely right here, however, your proposed solution won't work as this would strip "short user typed tags" as well. I will therefore see if I can specify the exact output of the rich editor to only strip those tags.